### DISCUSSION TITLE
stdout doesn't work?

### DISCUSSION THREAD

**Discussion by lazy-schemer** [autc04/Retro68#226] [Q&A] [answered]
I tried running a super simple 'hello world' program on System7, but I didn't see anything. 

```C
#include <stdio.h>
main () {
     printf ( "Hello, world!" ) ;
}
```

**Comment by autc04** [comment, marked as answer]
Does the HelloWorld sample program work?

It's probably a question of how you invoked the compiler:
By default, stdout will be implemented as a no-op, unless you link to the RetroConsole library.
If you are invoking the compiler directly, you will need to use `-lRetroConsole` to link to that library, if you're using CMake, add the `CONSOLE` option to `add_application` (as is done in the HelloWorld) sample.

**Reply by lazy-schemer** [reply]
thank you ! I was able to see the console output
